micrometer-tracing-bridges/micrometer-tracing-bridge-otel/src/main/java/io/micrometer/tracing/otel/bridge/OtelSpan.java

```diff
@@ -107,6 +107,24 @@ public Span tag(String key, String value) {
         return new OtelSpan(this.delegate);
     }
 
+    @Override
+    public Span tag(String key, long value) {
+        this.delegate.setAttribute(key, value);
+        return new OtelSpan(this.delegate);
```

Comment:
Come to think of it I wonder why wasn't I consistent here in comparison to other tracers. I'm creating a new `OtelSpan` back instead of returning `this`. :thinking:

Reply:
I see this patter in multiple places, I don't think that it's really required, but hard to be sure.